docs/xgcpolicy.md

```diff
@@ -173,13 +172,21 @@ The following options are ignored when specified with `-Xgcpolicy:balanced`:
 
         -Xgcpolicy:optavgpause
 
-: The "optimize for pause time" policy uses concurrent mark and concurrent sweep phases. Pause times are shorter than with `optthruput`, but application throughput is reduced because some garbage collection work is taking place while the application is running. Consider using this policy if you have a large heap size (available on 64-bit platforms), because this policy limits the effect of increasing heap size on the length of the garbage collection pause. However, if your application uses many short-lived objects, the `gencon` policy might produce better performance.
+: The "optimize for pause time" policy uses concurrent mark and concurrent sweep phases. Pause times are shorter than with `optthruput`, but application throughput is reduced because some garbage collection work is taking place while the application is running. 
```

Comment:
GC must run one way (STW) or another (concurrent) stealing CPU cycles, so that is not really a reason it's reducing throughput.
It's  because 1) mutator threads are penalized with doing GC work (for example execute object assess barriers) and 2) concurrent GC generate floating garbage (effectively dead objects that GC must keep alive), what in turn reduces effective free memory, which increases frequency of GCs.

Reply:
Wonder if those 'real' reasons are a bit too much detail for the user doc? Would it be better to just not go into detailed reasons? (i.e. just delete "...because some garbage collection work is taking place while the application is running.")
If you think those two reasons _should_ be included, is there some simpler wording that might work?

Reply:
how about: 
because some garbage collection work is taking place in the context of mutator (application) threads, and GC frequency is increased.

Reply:
Updated as suggested.